policyengine_us/variables/gov/states/ky/tax/income/credits/dependent_care_service/ky_dependent_care_service.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class ky_dependent_care_service(Variable):
```

Comment:
```suggestion
class ky_dependent_care_service_credit(Variable):
```

Reply:
rename file to `ky_dependent_care_service_credit.py`

Reply:
let's actually do `ky_cdcc.py` since that's basically what this is